sidebars.js

```diff
@@ -139,6 +139,7 @@ module.exports = {
             'platform-engineers/system-operation/vela-cli-image',
             'platform-engineers/system-operation/enable-addon-offline',
             'platform-engineers/system-operation/migration-from-old-version',
+            'platform-engineers/system-operation/system-info-diagnose',
```

Comment:
it can be merged with performance tunning doc, let call them:

"System Tunning"

Reply:
I don't think that this doc is related with "performance". At least it doesn't have much to do with "system tunning" 🤔

Reply:
But why do you think it's part of "Advanced Installation" ?

Reply:
> But why do you think it's part of "Advanced Installation" ?

It may not be the best place for this. But this may help to check the situation  when installing.